docs/incubating/identitycenter_group_membership.md

```diff
@@ -0,0 +1,72 @@
+---
+subcategory: "IAM Identity Center"
+---
+
+# huaweicloud_identitycenter_group_membership
+
+Manages an Identity Center group membership resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "identity_store_id" {}
```

Comment:
it's better to use data source to get the `identity_store_id`

Reply:
done